.github/workflows/prerelease_linux_on_demand.yml

```diff
@@ -49,9 +49,19 @@ jobs:
           username: ${{ env.DOCKER_HUB_ID }}
           password: ${{ env.DOCKER_HUB_PASSWORD }}
 
-      - name: Releasing all linux packages
+      - name: Preparing all linux packages
         run: make ci/prerelease/linux
 
+      - name: Generate checksum files
+        uses: ./.github/actions/generate-checksums
+        with:
+          files_regex: '.*\(tar.gz\|deb\|rpm\)'
+          files_path: 'dist'
+          run_sudo: true
+
```

Comment:
prerelease on-demand is not supposed to upload the packages to GH releases. Is this taken into account or am I loosing something? (I also see that `DISABLE_PUBLISH` is not used and it was used to control the upload to GH assets.